src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -227,7 +228,8 @@ public class FlowStateAllowedEventService {
                 HEARING_FEE_UNPAID,
                 HEARING_SCHEDULED,
                 EVIDENCE_UPLOAD,
-                migrateCase
+                migrateCase,
```

Comment:
I know migrateCase is not your state, however can you refactor this as it is clearly not formatted correctly

Reply:
Actually, I'd rather him leave this alone.

I'll be capturing this and other findings I have in another ticket and we'll do some cleanup as part of that.